### PR TITLE
AQCU-899: Ability to log different sides

### PR DESCRIPTION
**Pull request by zmoore-usgs**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ install:
   - Rscript -e 'install.packages(c("devtools", "testthat"))'
 
   #Install packages from github
-  - Rscript -e 'devtools::install_github("usgs-r/gsplot#469")'
+  - Rscript -e 'devtools::install_github("USGS-R/gsplot", ref="v0.8.1")'
```
Comment (lindsayplatt) [inline .travis.yml]: use PR number instead - #473

File: R/uvhydrograph-render.R [hunk 2]
```diff
@@ -180,47 +180,44 @@ createPrimaryPlot <- function(
...
   if((limsAndSides[['sides']][['reference']] == 4) || (limsAndSides[['sides']][['comparison']] == 4)){
-    plot_object <- lines(plot_object, x=0, y=0, side = 4, reverse = primarySeriesList[['inverted']]) %>%
+     plot_object <- lines(plot_object, x=0, y=0, side = 4, reverse = primarySeriesList[['inverted']]) %>%
         axis(side = 4, las = 0, reverse = primarySeriesList[['inverted']])
   }
   
+  #uncorrected data
+  if(!isEmptyOrBlank(primarySeriesList[['uncorrected']]) && !isEmptyVar(primarySeriesList[['uncorrected']][['points']])) {
+    plot_object <- plotTimeSeries(plot_object, primarySeriesList[['uncorrected']], "uncorrected", 
+                                  timezone, getPrimaryPlotConfig, list(uvInfo[['label']], limsAndSides$ylims[['primary']], limsAndSides$sides[['primary']]), excludeZeroNegativeFlag)
```
Comment (lindsayplatt) [inline R/uvhydrograph-render.R:190]: confused about how `doLog` ever makes it to `getPrimaryPlotConfig`. It should be in the list, but I see labels, limits, and sides added. Is it stored in `limsAndSides$sides[['primary']]`?

Comment (zmoore-usgs) [reply R/uvhydrograph-render.R:190]: The `doLog` that's used in `getPrimaryPlotConfig` is calculated within `plotTimeSeries` (utils-plot, line 194). 

When `logPrimary` is calculated (line 200) that is used for the section further down that limits other items being added on that axis to only having positive values (line 222)

Comment (lindsayplatt) [reply R/uvhydrograph-render.R:190]: Ohhhh there we go. I even looked at that already!

**Review by lindsayplatt — APPROVED** [review]
Looks good :+1:

**Pull request merged**